pipelines/tasks/rsem.wdl

```diff
@@ -19,7 +19,7 @@ task RsemExpression {
   output {
     File rsem_gene = "${rsem_out}.genes.results"
     File rsem_isoform = "${rsem_out}.isoforms.results"
-    File rsem_gene_count = "${rsem_out}.gene.expected_counts"
+    ## File rsem_gene_count = "${rsem_out}.gene.expected_counts"
```

Comment:
Why did you choose to drop this? What was it previously doing?

Reply:
If it's really being dropped, should it be taken out vs commented out?